src/sentry/static/sentry/app/components/group/inboxBadges/times.tsx

```diff
@@ -17,7 +17,7 @@ type Props = {
   firstSeen: string;
 };
 
-const TimesBadge = ({lastSeen, firstSeen}: Props) => {
+const Times = ({lastSeen, firstSeen}: Props) => {
```

Comment:
I think i had renamed to `<TimesBadge>` because we have a `<Times>` component that looks nearly identical. I see that the folder is named badges already. hmmmm Could be TimesTag or something. I dunno